### ISSUE TITLE
I cant see the folders and files in Plex

### ISSUE THREAD

**Issue by vampywiz17** [yadayada/acd_cli#139]
Hello,

I mount the amazon Cloud drive, it is working well, i see the files in file manager and i able to open (stream) it. But if i install the plex server and try to add the mounted drive, inside plex, it is empty...

What i do it wrong? (the op. system is Lubuntu and python 3.4)

**Comment by jamesrascal** [comment]
Is your plex service running as the same user as your acd mount? Do you have the option on to allow other users ?



Kind Regards,Roger Gentry

**Comment by jamesrascal** [comment]
Try using this as the mount command and is should correct the issue: acd_cli --allow-other

**Comment by vampywiz17** [comment]
Working!!

Thanks a lot!!

**Comment by vampywiz17** [comment]
Last question:

How to set that the mount script running every start in lubuntu?

**Comment by yadayada** [comment]
You can use a crontab entry.

**Comment by HazCod** [comment]
@vampywiz17 Sorry to revive this, but how did you get your posters etc to load correctly from acd_cli ? I have a show.jpg but plex doesn't load it so it seems.

**Comment by vampywiz17** [comment]
Well, i use the "Agents" that use the covers, etc... it is working well, not necessary to use posters manually...

**Comment by rpernavas** [comment]
Sorry to hijack this thread, but I've followed a plex installation guide and everything works great for a while and then I get authentication errors.  I am a software dev so familiar with whatever you want from me, but is there a known issue with this happening?  I admit I am currently far too busy with work to pore through the documentation.  I just followed the guide.  I know Amazon had problems recently, but these seem to be sorted, I managed to watch half of Commando last night before having to reboot the server and run acd_cli sync and remount the drive.
